target/linux/ar71xx/base-files/etc/board.d/02_network

```diff
@@ -479,6 +479,9 @@ ar71xx_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0@eth0" "2:lan" "3:wan"
 		;;
+	xd6700)
+		ucidef_set_interfaces_lan_wan "eth1" "eth0"
+		;;
```

Comment:
Can you please move this case between the `wzr-hp-g450h` and the `z1` one to maintain alphabetical order?